### ISSUE TITLE
([]["isPrototypeOf"] + "")[11]

### ISSUE THREAD

**Issue by Andromeda606** [aemkei/jsfuck#123]
I don't know if anyone has said that, but this gives the result P.
```JS
([]["isPrototypeOf"] + "")[11]
```

**Comment by hazzik** [comment]
@Andromeda606 do you realize that “isPrototypeOf” requires “P” to get that word?

**Comment by hazzik** [comment]
> the same is heard here.

No, it is completely different: `“undefined”` is an “atomic” construct that does not require any letters — `[][[]]+[]`.

**Comment by Andromeda606** [comment]
> > aynısı burada da duyuluyor.
> 
> Hayır, tamamen farklıdır: `“undefined”`herhangi bir harf gerektirmeyen “atomik” bir yapıdır — `[][[]]+[]`.
I understand.
